packages/@aws-cdk/aws-batch-alpha/test/ecs-container-definition.test.ts

```diff
@@ -711,4 +713,50 @@ describe('Fargate containers', () => {
       },
     });
   });
+
+  test('can set ephemeralStorage', () => {
+    // WHEN
+    new EcsJobDefinition(stack, 'ECSJobDefn', {
+      container: new EcsFargateContainerDefinition(stack, 'EcsFargateContainer', {
+        ...defaultContainerProps,
+        fargatePlatformVersion: ecs.FargatePlatformVersion.LATEST,
+        ephemeralStorageSize: Size.gibibytes(100),
+      }),
+    });
+
+    // THEN
+    Template.fromStack(stack).hasResourceProperties('AWS::Batch::JobDefinition', {
+      ...pascalCaseExpectedProps,
+      ContainerProperties: {
+        ...pascalCaseExpectedProps.ContainerProperties,
+        ExecutionRoleArn: {
+          'Fn::GetAtt': ['EcsFargateContainerExecutionRole3286EAFE', 'Arn'],
+        },
+        EphemeralStorage: {
+          SizeInGiB: Size.gibibytes(100).toGibibytes(),
+        },
+      },
+    });
+  });
+
+  test('ephemeralStorage validation', () => {
```

Comment:
you will also need to consider the case where ephemeral storage is supplied as a token.